### DISCUSSION TITLE
Abnormal style attribute causes Livewire mount() twice

### DISCUSSION THREAD

**Discussion by kylehon** [livewire/livewire#4841] [7. Bugs (archived)]
Codes to reproduce:

Env:
```
        "livewire/livewire": "^2.5",
        "laravel/framework": "^9.0",
```

blade:
```blade
<section style="background-image: url('');">
    <livewire:anycomponent />
</section>
```

Anycomponent.php
```php
<?php
namespace App\Http\Livewire;

use Livewire\Component;

class Anycomponent extends Component
{
    public function mount()
    {
        echo 'mount';
        \Log::debug('log mount');
    }
}
```

HTML output:
`mount`
It is okay to output 1 time only.

However, in log file,
```
log mount
log mount
```
It logged twice, which means mount() is actually run twice.

The bug is caused by 
```html
<section style="background-image: url('');">
```
❌
Same time happens if the code is 
```html
<section style="background-image: url();">
```
❌

----

If the line changes to 
```html
<section style="background-image: url(a);">
```
or
```html
<section style="background-image: url('a');">
```
It works normally.

----

**Comment by joshhanley** [comment]
@desket thanks for reporting! Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

**Comment by mauriciv** [comment]
By any chance, did you try using the `@livewire` syntax instead of `<livewire:> `? 

I'm finding that, with nested components, using the former `mount` only gets called once, but using the later makes `mount` get called on subsequent interactions

**Reply by kylehon** [reply]
not sure but was using `<livewire:>`

If what you describe is true, more likely it would like a Laravel issue instead of Livewire directly?
